common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgBlueprintStore.java

```diff
@@ -241,10 +242,15 @@ private void handleBlueprintImport(final File bpDir, final Map<String, VraNgBlue
 		VraNgBlueprint bp = loadBlueprintFromFilesystem(bpDir);
 		String bpID;
 
+		// Check the blueprint folder name(bpName) with the name from details.json(bp.getName())
+		if (!Objects.equals(bp.getName(), bpName)) {
```

Comment:
If it comes to bp.getName() = null (if in details.json -> "name": null), then it would make NullPointException. 
But instead of that we can use "!bpName.equals(bp.getName())" hoping that folder-name would never be null. 
But to be on the safe side, Objects.equals() might be best. WDYT?

Reply:
Ahh cool stuff! Ty